examples/mobile/UIComponents/index.html

```diff
@@ -56,11 +56,6 @@ <h1>
 						Text fields
 					</a>
 				</li>
-				<li class="ui-li-anchor">
-					<a href="components/toast/index.html">
-						Toast
-					</a>
-				</li>
 				<li class="ui-li-anchor">
```

Comment:
Navigation Elements are not part of current guide. Some of items in it should became part of View controls in future.